x/staging/posts/types/keys.go

```diff
@@ -108,12 +108,17 @@ func UserAnswersByPostPrefix(id string) []byte {
 	return append(UserAnswersStorePrefix, []byte(id)...)
 }
 
-// UserAnswersStoreKey returns the store key used to store the user answer containing the given data
+// UserAnswersStoreKey returns the prefix used to store all the reports for the post having the given id
 func UserAnswersStoreKey(id, user string) []byte {
 	return append(UserAnswersByPostPrefix(id), []byte(user)...)
 }
 
-// ReportStoreKey turns an id into a key used to store a report inside the reports store
-func ReportStoreKey(id string) []byte {
+// ReportsByPostIDPrefix turns an id into a key used to store a report inside the reports store
```

Comment:
```suggestion
// ReportsByPostIDPrefix returns a prefix used to store all the reports for the post having given id
```
I think the function returns a prefix, not a key.